src/ubuntu/build-scripts/build-rootfs.sh

```diff
@@ -49,12 +49,12 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-echo "Checking existence of /rootfs/$arch/bin"
+echo "Checking existence of /rootfs/$arch/usr/bin"
 docker exec $buildRootFSContainer \
-    [ -d /rootfs/$arch/bin ]
+    [ -d /rootfs/$arch/usr/bin ]
 
 if [ $? -ne 0 ]; then
-    echo "Rootfs build failed: rootfs/$arch/bin empty"
+    echo "Rootfs build failed: rootfs/$arch/usr/bin empty"
```

Comment:
It looks like these changes cause the build to fail for ubuntu/18.04/cross/freebsd/11.  I would recommend defining a variable for the bin path that defaults to `/rootfs/$arch/bin` but can be overridden by a parameter to this script. And then override it in the src/ubuntu/18.04/cross/armel-tizen/hooks/pre-build script by passing `rootfs/armel/usr/bin`.

Reply:
Thank you for the review I will fix it.